pkg/backup/backup.go

```diff
@@ -244,6 +248,17 @@ func (kb *kubernetesBackupper) BackupWithResolvers(
 	log.Infof("Including namespaces: %s", backupRequest.NamespaceIncludesExcludes.IncludesString())
 	log.Infof("Excluding namespaces: %s", backupRequest.NamespaceIncludesExcludes.ExcludesString())
 
+	// check if there are any namespaces included in the backup which are managed by argoCD
+	// We will check for the existence of a ArgoCD label in the includedNamespaces and add a warning
+	// so that users are at least aware about the existence of argoCD managed ns in their backup
```

Comment:
Could you add references to the issues that argo-cd and velero don't work together, so that if such issues are resolved we know we can remove the warning?

Reply:
Not a bad idea :D

Reply:
@shubham-pampattiwar WDYT?

Reply:
Done, updated the PR.